src/test/java/org/killbill/billing/plugin/stripe/TestBase.java

```diff
@@ -67,7 +101,7 @@ public void setUp() throws Exception {
         context = Mockito.mock(CallContext.class);
         Mockito.when(context.getTenantId()).thenReturn(UUID.randomUUID());
 
-        account = TestUtils.buildAccount(DEFAULT_CURRENCY, DEFAULT_COUNTRY);
+        account = buildAccount(DEFAULT_CURRENCY, DEFAULT_COUNTRY);
```

Comment:
Instead of overriding `buildAccount`, could you just re-mock the email? i.e. would this work:

```
account = TestUtils.buildAccount(DEFAULT_CURRENCY, DEFAULT_COUNTRY);
Mockito.when(account.getEmail()).thenReturn(UUID.randomUUID().toString() + "@example.com");
```

Reply:
Yes, that worked. Much better now